### PR TITLE
WIP: Add CI config for openshift/ironic-image.

### PR DESCRIPTION
**Pull request by russellb**

### CODE REVIEW

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```
Comment (juliakreger) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: Is the tag correct? Not go-lang artifact...

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  to: ironic
```
Comment (juliakreger) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: I think this should be ironic-image

Comment (russellb) [reply ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: k, will try that

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml (hunk 1 above)
Comment (juliakreger) [reply ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: Disregard, same in ipa image downloader

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
```
Comment (juliakreger) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: So in the downloader, this was base-os

https://github.com/openshift/release/pull/4411/files

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
```
Comment (juliakreger) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: https://github.com/openshift/release/pull/4411/files shows "base-os"

File: ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  to: ironic-image
+promotion:
+  name: "4.2"
+  namespace: ocp
+resources:
```
Comment (juliakreger) [inline ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml]: So I think we're going to update the repos: see https://github.com/openshift/release/pull/4411/files raw_steps definition.

**Comment by elfosardo** [comment]
this is duplicate of #4348

**Pull request closed without merging**